sermant-integration-tests/xds-service-test/xds-service-discovery/xds-service-discovery-plugin/src/main/java/io/sermant/xds/service/discovery/TemplateDeclarer.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package io.sermant.xds.service.discovery;
+
+import io.sermant.core.plugin.agent.declarer.AbstractPluginDeclarer;
+import io.sermant.core.plugin.agent.declarer.InterceptDeclarer;
+import io.sermant.core.plugin.agent.matcher.ClassMatcher;
+import io.sermant.core.plugin.agent.matcher.MethodMatcher;
+
+/**
+ * template bytecode enhancement declarer
+ *
+ * @author daizhnyu
+ * @since 2024-07-05
+ */
+public class TemplateDeclarer extends AbstractPluginDeclarer {
+    @Override
+    public ClassMatcher getClassMatcher() {
+        // 匹配需要拦截的类，通过全限定名匹配
```

Comment:
Notes should be in English as much as possible

Reply:
done